### PR TITLE
Chat UI

### PR DESCRIPTION
**Pull request by khumair**

### CODE REVIEW

File: src/Routes.tsx [hunk 1]
```diff
@@ -19,6 +20,7 @@ const Routes = () => (
     <Route exact path="/reset-password" component={PWReset} />
     <Route exact path="/jobseeker/profile" component={Profile} />
     <Route exact path="/company/profile" component={CompanyMatch} />
+    <Route exact path="/company/profile1" component={CompanyHomePage} />
```
Comment (Wyfy0107) [inline src/Routes.tsx:23]: I believe these should be one page

**Review by Tsaikkari — CHANGES_REQUESTED** [review]
You have all the old code there, so you haven't pulled the latest code.

File: src/components/JobPost/Tags.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+//             <input
+//     type="text"
+//     onKeyUp={event => addTags(event)}
+//     placeholder="Press enter to add tags"
+// />
+//         </div>
+<>
+hello
+</>
+
```
Comment (Wyfy0107) [inline src/components/JobPost/Tags.tsx:30]: If you're not using this component then you should remove it

File: src/components/JobPost/index.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    const jobPosted = {
+      jobTitle,
+      jobDescription,
+      city,
+      country,
+      skills,
+    }
+    // dispatch(creatingJobPostRequest(jobPosted))
```
Comment (Wyfy0107) [inline src/components/JobPost/index.tsx:39]: Why commented ?

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

**Comment by Tsaikkari** [comment]
@khumair I added your chatbox component to my pr with latest code. You can make a new branch and pull latest code from develop into it. After the pull, check out that you don't have any extra folders/files in it. Compare the team4 develop branch file structure to yours. Don't use this ChatUI branch anymore when you make a pr, because all the old code is there and we don't want to merge them. 

When you have pulled the new code to the new branch, make yet another branch and work on there, so that you have the previous branch "clean". Make a new pr as soon as you've made a new feature.

File: src/components/JobPost/index.tsx (hunk 3 above)
Comment (khumair) [reply src/components/JobPost/index.tsx:39]: it has some error , 
but now its fixed

**Comment by Wyfy0107** [comment]
@Tsaikkari Why did you reopen this pr, didn't you merge @khumair code to the previous pr ?

**Comment by Tsaikkari** [comment]
@Wyfy0107 I copied and pasted the chatbox component into my repo and closed the pr, but I don't think it's a good and fair way to go. It's better to give him the chance to fix his own errors and learn properly how to use git and github in a team. I don't want him to miss that opportunity.

**Pull request closed without merging**